graph-refresh/overlays/test/cronjob.yaml

```diff
@@ -28,6 +28,8 @@ spec:
               env:
                 - name: DEBUG_LEVEL
                   value: "0"
+                - name: SUBCOMMAND
```

Comment:
I think as you provided `SUBCOMMAND`, then you should add it in `app.sh` like in https://github.com/thoth-station/package-releases-job/blob/master/app.sh for https://github.com/thoth-station/graph-refresh-job/blob/master/app.sh, WDYT @pacospace ?

Reply:
Ah you are right, we dont need subcommand as graph refresh job is always a producer.

Reply:
I'd add it as @pacospace advised to do so in case of `package-releases-job`, PTAL https://github.com/thoth-station/thoth-application/pull/425#discussion_r488812289.

Reply:
Hmm, the graph-refresh-jobs app.sh doesn't have the SUBCOMMAND variable  - 
https://github.com/thoth-station/graph-refresh-job/blob/28998f7d6e4b113206cdd8c040c1fcdd6d803a3b/app.sh#L20